snapcraft/cli/lifecycle.py

```diff
@@ -135,9 +140,65 @@ def _execute(  # noqa: C901
     return project
 
 
+def _run_pack(snap_command: List[str]) -> str:
+    ret = None
+    stdout = ""
+    stderr = ""
+    with subprocess.Popen(
+        snap_command, stdout=subprocess.PIPE, stderr=subprocess.PIPE
+    ) as proc:
+        if indicators.is_dumb_terminal():
+            echo.info("Snapping...")
+            ret = proc.wait()
+        else:
+            message = f"\033[0;32mSnapping \033[0m"
+            progress_indicator = progressbar.ProgressBar(
+                widgets=[message, progressbar.AnimatedMarker()], maxval=7
```

Comment:
I assume we don't know actual progress here... do one of these progress bar options simply "bounce"?  Then drop the whole count logic below?

Reply:
We don't and much less now using snap pack. I have moved to a more elegant solution for counting.